### DISCUSSION TITLE
Bulk upload emojis

### DISCUSSION THREAD

**Discussion by germainelee** [owncast/owncast#4041] [Ideas] [answered]
Currently, I can upload one custom emoji image at a time. This is troublesome when there is a pack of 10+ emojis I'd like to use from [emoji.gg](https://emoji.gg/packs).

It would be nice if I could select multiple/all images at once, by pressing Ctrl/Cmd+A or using the Shift key or Ctrl key.

**Comment by gabek** [comment, marked as answer]
You can bulk add them by placing them in the `data/emoji` directory.
https://owncast.online/docs/chat/emoji/#manual-management

**Reply by germainelee** [reply]
Oh that's nice! I wasn't aware. Should there be a small text that links to its respective documentation? Something like "How to upload emoji packs. [Learn more](https://owncast.online/docs/chat/emoji/#manual-management)"

**Reply by gabek** [reply]
We do that elsewhere, so that would be a good idea.

**Reply by germainelee** [reply]
Awesome! I've just raised #4042 for this.
